### PR TITLE
added silhouette-wallpaper

### PR DESCRIPTION
**Pull request by p01arst0rm**
## Purpose

new wallpapers!
This work is licensed under a Creative Commons 
Attribution 4.0 International License.

JIRA issue: [CORE-13000](https://jira.reactos.org/browse/CORE-13000)

## Proposed changes
 
* added modules/wallpapers/reactos_silhouette_light.png
* added modules/wallpapers/reactos_silhouette_dark.png
* updated ReadMe.txt

## TODO
no TODO, unless feedback is given

### CODE REVIEW

File: modules/wallpapers/ReadMe.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@ ReactOS_CirclesBlue_2.jpg           | See above
 ReactOS_CirclesGold_1.jpg           | See above                                      | See above
 ReactOS_CirclesGold_2.jpg           | See above                                      | See above
 ReactOS_Hexagons.jpg                | See above                                      | http://www.reactos.org/forum/viewtopic.php?p=109866#p109866
+Silhouette collection				| Jake Collins (polar@ever3st.com)				 | https://ever3st.com/Graphics/Loot/
```
Comment (binarymaster) [inline modules/wallpapers/ReadMe.txt]: Please use spaces instead of tabs here. That change was ok in previous PR.

Also why don't you visit our chat? We could help you with git. :slightly_smiling_face:

Comment (p01arst0rm) [reply modules/wallpapers/ReadMe.txt]: which chat? and changing now

**Comment by MailYouLater** [comment]
> This work is licensed under a Creative Commons
> Attribution-NoDerivatives 4.0 International License.

Is [CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/) compatible with the ReactOS license? If I remember correctly, ReactOS is under GPL2, and I thought GPL2 required that distribution of modified versions be allowed, which "No Derivatives" prohibits.

Also, you can `git rebase -i` to interactively reorganize and squash commits, then `git push --force` to the branch on GitHub that's associated with the PR. That way you can clean up your commit history without having to open a new PR.

**Comment by p01arst0rm** [comment]
> > This work is licensed under a Creative Commons
> > Attribution-NoDerivatives 4.0 International License.
> 
> Is [CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/) compatible with the ReactOS license? If I remember correctly, ReactOS is under GPL2, and I thought GPL2 required that distribution of modified versions be allowed, which "No Derivatives" prohibits.
> 
> Also, you can `git rebase -i` to interactively reorganize and squash commits, then `git push --force` to the branch on GitHub that's associated with the PR. That way you can clean up your commit history without having to open a new PR.

it isnt a derivative work, as far as i am concerned? it is an original piece of art

**Comment by p01arst0rm** [comment]
> > > This work is licensed under a Creative Commons
> > > Attribution-NoDerivatives 4.0 International License.
> > 
> > 
> > Is [CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/) compatible with the ReactOS license? If I remember correctly, ReactOS is under GPL2, and I thought GPL2 required that distribution of modified versions be allowed, which "No Derivatives" prohibits.
> > Also, you can `git rebase -i` to interactively reorganize and squash commits, then `git push --force` to the branch on GitHub that's associated with the PR. That way you can clean up your commit history without having to open a new PR.
> 
> it isnt a derivative work, as far as i am concerned? it is an original piece of art

ah, you mean if someone wishes to modify the wallpapers. i can use the derivative version.

**Comment by p01arst0rm** [comment]
license updated

**Comment by MailYouLater** [comment]
> > > This work is licensed under a Creative Commons
> > > Attribution-NoDerivatives 4.0 International License.
> > 
> > Is [CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0/) compatible with the ReactOS license? If I remember correctly, ReactOS is under GPL2, and I thought GPL2 required that distribution of modified versions be allowed, which "No Derivatives" prohibits.
>
> it isnt a derivative work, as far as i am concerned? it is an original piece of art

What I mean is that a GPL project is *required* to allow others to make modifications. If you choose CC BY-ND, you're disallowing that, so I'm not sure ReactOS can accept it into the main repository.
I'm not a lawyer, so I don't know all the ins and outs of this stuff, but I think it would be more likely that it would be added if you chose a license that doesn't disallow derivative works.

Basically, if I wanted to, the GPL allows me to fork the project, change the name, and go through all the included images replacing the ReactOS logo with my own, and distribute that (with source), but CC BY-ND says I'm not allowed to distribute that because my new version of this wallpaper that has my logo instead of the ReactOS logo is derived from yours. See the problem.

**Comment by MailYouLater** [comment]
> license updated

Oops, sorry, I hadn't seen your new post yet when I posted mine.
(I should probably learn to type faster.)

**Comment by p01arst0rm** [comment]
> > license updated
> 
> Oops, sorry, I hadn't seen your new post yet when I posted mine.
> (I should probably learn to type faster.)

no problem! i misunderstood, but changed now.
